### DISCUSSION TITLE
Pregunta

### DISCUSSION THREAD

**Discussion by mauricio2093** [mauricio2093/tensorflowJS#2] [Q&A] [answered]
Tengo una pregunta referente a esta parte del código
cuando corro el programa de  AI-Intro-TFJS
`node index.js image1.png`
no encuentra la imagen muchas gracias por su ayuda.

**Comment by H4ckR0B0t** [comment, marked as answer]
Mira bien la extensión de tu archivo de imagen es .jpg
`node index.js image1.jpg`

**Reply by mauricio2093** [reply]
Gracias!
